### PR TITLE
feat(start) allow overriding inspector on start

### PR DESCRIPTION
**Pull request by jrwpatterson**
Signed-off-by: jrwpatterson <jrwpatterson@gmail.com>## Hey, I just made a Pull Request!

Hey I've made a small change to allow adding changes to the inspect flag on the backend start. 

We run our app out of `docker-compose` and it wants to listen to the debugger on `ws://127.0.0.1:9229` this isn't compatible with docker compose and remote debugging. I did think about moving it to the listener but then I decided that might upset current work flows but also wouldn't allow for things like custom port listening. 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/brown-cameras-marry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Allows the ability to override the inspect and add after the `=` so you can now change the i.p. and port of the inspect and inspectBrk by adding for example `--inspect=0.0.0.0:9229`
```
Comment (jhaals) [inline .changeset/brown-cameras-marry.md:5]:
```suggestion
Adds the ability to specify the listen address for the `--inspect` and `--inspect-brk` command. You can now set the ip and port of the `inspect` and `inspectBrk` by adding for example `--inspect=0.0.0.0:9229`
```

File: .github/vale/Vocab/Backstage/accept.txt [hunk 2]
```diff
@@ -448,3 +448,4 @@ zoomable
 zsh
 Pulumi
 Lightsail
+inspectBrk
```
Comment (jhaals) [inline .github/vale/Vocab/Backstage/accept.txt:451]: We can avoid this line by adding placing `inspectBrk` in backticks :) 
```suggestion
```

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks for the PR! This seems like a good change, just a minor tweak to the changeset and accept.txt and I think we're good to go.

**Comment by jrwpatterson** [comment]
Thanks @jhaals done signed off and committed both. don't know why I didn't think of that!

File: plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts [hunk 3]
```diff
@@ -15,38 +15,37 @@
  */
 import 'buffer';
 
-import { errorHandler, getVoidLogger } from '@backstage/backend-common';
```
Comment (jhaals) [inline plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts:18]: oh I think some unrelated changes snuck in somehow!

Comment (jrwpatterson) [reply plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts]: yup @jhaals sorry was trying to see whats causing the error on the Kube tests, they seem to be happening across all branches that have pulled from main today... 😢 

but reverted

Comment (jrwpatterson) [reply plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts]: but the test errors are blocking me and I can't replicate locally

**Comment by jrwpatterson** [comment]
@jhaals now the stars are aligned an its finally build 😆 I'm hoping we can get it in before it breaks again! 🤞

**Review by jhaals — APPROVED** [review]
Looking good, thank you 👏
